packages/blade/src/components/ListView/ListViewFilters.web.tsx

```diff
@@ -0,0 +1,58 @@
+import { useState } from 'react';
+import type { ListViewFilterProps } from './types';
+import { makeAnalyticsAttribute } from '~utils/makeAnalyticsAttribute';
+import BaseBox from '~components/Box/BaseBox';
+import { MetaConstants, metaAttribute } from '~utils/metaAttribute';
+import { FilterIcon } from '~components/Icons';
+import { Button } from '~components/Button';
+import { Counter } from '~components/Counter';
+import { Box } from '~components/Box';
+
+const ListViewFilters = ({
+  testID,
+  children,
+  quickFilters,
+  onSearchChange,
+  numberOfSelectedFilters,
```

Comment:
Not a requirement but I was wondering if we have any easy way to figure out number of selected filters internally using context because we will have FilterChipGroup inside this 🤔